### PR TITLE
Update plagiarism_detection_es.ipynb

### PR DESCRIPTION
**Pull request by priscillaparodi**
updated to latest eland version

### CODE REVIEW

**Comment by priscillaparodi** [comment]
https://github.com/elastic/elasticsearch-labs/issues/144

File: supporting-blog-content/plagiarism-detection-with-elasticsearch/plagiarism_detection_es.ipynb [hunk 1]
```diff
@@ -91,7 +91,7 @@
         "# Set the model name from Hugging Face and task type\n",
         "# open ai detector model - developed by open ai https://github.com/openai/gpt-2-output-dataset/tree/master/detector\n",
         "hf_model_id='roberta-base-openai-detector'\n",
-        "tm = TransformerModel(hf_model_id, \"text_classification\")\n",
+        "tm = TransformerModel(model_id=hf_model_id, task_type="text_classification")\n",
```
Comment (miguelgrinberg) [inline supporting-blog-content/plagiarism-detection-with-elasticsearch/plagiarism_detection_es.ipynb:94]: The quotes are not escaped here

File: supporting-blog-content/plagiarism-detection-with-elasticsearch/plagiarism_detection_es.ipynb [hunk 2]
```diff
@@ -121,7 +121,7 @@
         "# Set the model name from Hugging Face and task type\n",
         "# sentence-transformers model\n",
         "hf_model_id='sentence-transformers/all-mpnet-base-v2'\n",
-        "tm = TransformerModel(hf_model_id, \"text_embedding\")\n",
+        "tm = TransformerModel(model_id=hf_model_id, task_type="text_embedding")\n",
```
Comment (miguelgrinberg) [inline supporting-blog-content/plagiarism-detection-with-elasticsearch/plagiarism_detection_es.ipynb:124]: This one too, the quotes need to be escaped.

Comment (priscillaparodi) [reply supporting-blog-content/plagiarism-detection-with-elasticsearch/plagiarism_detection_es.ipynb]: updated. thank you.

**Review by miguelgrinberg — APPROVED** [review]

**Pull request merged**